specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-04-01-preview/workspaceRP.json

```diff
@@ -0,0 +1,6198 @@
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrivateLinkResource"
+          }
+        }
+      }
+    },
+    "PrivateLinkResourceProperties": {
+      "description": "Properties of a private link resource.",
+      "type": "object",
+      "properties": {
+        "groupId": {
+          "description": "The private link resource group id.",
+          "type": "string"
+        },
+        "requiredMembers": {
+          "description": "The private link resource required member names.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "requiredZoneNames": {
+          "description": "The private link resource Private link DNS zone name.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "PrivateLinkServiceConnectionState": {
+      "description": "A collection of information about the state of the connection between service consumer and provider.",
+      "type": "object",
+      "properties": {
+        "actionsRequired": {
+          "description": "Some RP chose \"None\". Other RPs use this for region expansion.",
+          "type": "string"
+        },
+        "description": {
+          "description": "User-defined message that, per NRP doc, may be used for approval-related message.",
+          "type": "string"
+        },
+        "status": {
+          "$ref": "#/definitions/PrivateEndpointServiceConnectionStatus"
+        }
+      }
+    },
+    "ProvisioningState": {
+      "description": "The current deployment state of workspace resource. The provisioningState is to indicate states for resource provisioning.",
+      "enum": [
+        "Unknown",
+        "Updating",
+        "Creating",
+        "Deleting",
+        "Succeeded",
+        "Failed",
+        "Canceled"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "ProvisioningState",
+        "modelAsString": true
+      }
+    },
+    "PublicNetworkAccessType": {
+      "description": "The public network access flag.",
+      "enum": [
+        "Enabled",
+        "Disabled"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "PublicNetworkAccessType",
+        "modelAsString": true
+      }
+    },
+    "RaiBlocklistConfig": {
+      "description": "Azure OpenAI blocklist config.",
+      "type": "object",
+      "properties": {
+        "blocking": {
+          "type": "boolean"
+        },
+        "blocklistName": {
+          "type": "string"
+        }
+      }
+    },
+    "RaiPolicyContentFilter": {
+      "description": "Azure OpenAI Content Filter.",
+      "type": "object",
+      "properties": {
+        "allowedContentLevel": {
+          "$ref": "#/definitions/AllowedContentLevel"
+        },
+        "blocking": {
+          "type": "boolean"
```

Comment:
Same here

Reply:
same as above, will add description but hope we can keep the boolean format